internal/operands/template-validator/resources.go

```diff
@@ -16,38 +16,30 @@ import (
 //        repository, and import it as a go module
 
 const (
-	containerPort         = 8443
-	kubevirtIo            = "kubevirt.io"
-	secretName            = "virt-template-validator-certs"
-	virtTemplateValidator = "virt-template-validator"
-
-	ServiceAccountName = "template-validator"
-	ServiceName        = virtTemplateValidator
-	DeploymentName     = virtTemplateValidator
+	containerPort          = 8443
+	kubevirtIo             = "kubevirt.io"
+	secretName             = "virt-template-validator-certs"
+	virtTemplateValidator  = "virt-template-validator"
+	ClusterRoleName        = "template:view"
+	ClusterRoleBindingName = "template-validator"
+	WebhookName            = "virt-template-validator"
+	ServiceAccountName     = "template-validator"
+	ServiceName            = virtTemplateValidator
+	DeploymentName         = virtTemplateValidator
 )
 
 var commonLabels = map[string]string{
 	kubevirtIo: virtTemplateValidator,
 }
 
-func ClusterRoleName(namespace string) string {
-	const clusterRoleName = "template:view"
-	return clusterRoleName + "-" + namespace
-}
-
-func ClusterRoleBindingName(namespace string) string {
-	return ServiceAccountName + "-" + namespace
-}
-
-func ValidatingWebhookName(namespace string) string {
-	return virtTemplateValidator + "-" + namespace
-}
-
 func newClusterRole(namespace string) *rbac.ClusterRole {
 	return &rbac.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   ClusterRoleName(namespace),
-			Labels: commonLabels,
+			Name:      ClusterRoleName,
+			Namespace: namespace,
```

Comment:
This is a cluster role, so the namespace needs to be `""`.

Reply:
Done